src/components/Panel/Channels/index.vue

```diff
@@ -176,6 +180,20 @@ export default {
     },
   },
 
+  watch: {
+    groupChannels (newChannels, oldChannels) {
+      if (newChannels.length > oldChannels.length) {
+        const memberIDs = new Set()
+        newChannels.forEach(({ members }) => {
+          members.reduce((s, e) => s.add(e), memberIDs)
+        })
+        if (memberIDs.size > 0) {
```

Comment:
This check can be a responsibility of `fetchUsers` (it already is, so this is not needed)